components/VC/common/VCUtils.tsx

```diff
@@ -328,3 +335,34 @@ export class Display {
     return this.backgroundImage ?? defaultBackgroundImage;
   }
 }
+
+export function getIssuerAuthenticationAlorithmForMdocVC(
```

Comment:
We can also move this to the OpenId4vp plugin and return along with unSignedVPToken for the calling application to easily sign and return back